jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/file/AssetFileGenerator.java

```diff
@@ -689,54 +740,62 @@ public void generateFiles(List<SubscriptionVH> subscriptions, String filePath) {
 					return;
 				}
 				try {
-
+					targetTypeList.add("subscription");
 					FileManager
 							.generateSubscriptionFiles(subscriptionInventoryCollector.fetchSubscriptions(subscription));
 					log.info("subscription data saved!");
```

Comment:
Since we are already making changes here and they are related to log - let's get rid of such screams. Remove ```!``` in log messages. It doesn't help.

Reply:
done.